docs/contributing/code-contribution-guide.md

```diff
@@ -270,11 +270,11 @@ type Something = {
 };
 ```
 
-### 2. Use an `interface` when functions have object arguments
+### 2. Use an `interface` when functions have object arguments or returns
```

Comment:
```suggestion
### 2. Use an interface when functions take an object as an argument or when functions return an object
```

Suggest changing to: "Use an interface when functions take an object as an argument or when functions return an object" - also change at line 252 above